PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/sparkserverless/SparkServerlessClusterOpsCtrl.java

```diff
@@ -80,6 +82,25 @@ public SparkServerlessClusterOpsCtrl(@NotNull SparkServerlessClusterOps sparkSer
                             pair.getRight(), pair.getLeft());
                     updateDialog.show();
                 }, ex -> LOG.error(ex.getMessage(), ex));
+
+        this.sparkServerlessClusterOps.getSubmitAction()
+                .observeOn(ideSchedulers.dispatchUIThread())
+                .subscribe(clusterNodePair -> {
+                    LOG.info(String.format("Submit message received. cluster: %s, node: %s",
+                            clusterNodePair.getLeft(), clusterNodePair.getRight()));
+
+                    try {
+                        ServerlessSparkRunConfigurationSubmitter submitter =
+                                new ServerlessSparkRunConfigurationSubmitter(
+                                        (Project) clusterNodePair.getRight().getProject(),
+                                        clusterNodePair.getLeft().getName());
+
+                        submitter.submit();
+
+                    } catch (Exception ex) {
+                        LOG.error(ex);
+                    }
+                }, ex -> LOG.error(ex.getMessage(), ex));
```

Comment:
We can get this in first. However, the context menu in HDInsight project is still broken.

Probably we can do the following:
1. Add submit action to HDI node as well
2. Create two project template, Spark pool and HDInsight
3. Depend on project type, start "ServerlessRunConfigurtion" or "HDI runconfiguratin"

With this, context menu submission will work for both HDI and Cosmos Spark pool